Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml

```
@@ -0,0 +1,14 @@
+- type: entity #Delta-V safety glasses
```

```suggestion
- type: entity
```
No need for a change marker since it is in a DeltaV file.